templates/docs/patterns/tabs/index.md

```diff
@@ -35,6 +35,21 @@ When you need to group a number of related blocks of content within an area on t
 View example of the tabs content pattern
 </a></div>
 
+
+## Theming
+
+Tabs use Vanilla's light theme by default. There are two ways to switch between the light and the dark themes:
+
+- Change the default: go to `_settings_themes.scss` and set `$theme-default-p-tabs` to `dark`
```

Comment:
Do we want to use bulleted list here? They will not have our preferred "divided" style, just a deprecated native took. I've been avoiding this recently.

Otherwise to make it look as expected we would need to write it in HTML instead of markdown.

Reply:
good point, addressed